test-cases/longevity/longevity-all-compressions-3days.yaml

```diff
@@ -0,0 +1,50 @@
+test_duration: 4320
+prepare_write_cmd: [ "cassandra-stress write cl=QUORUM n=50000000 -schema 'replication(factor=3) compression=LZ4Compressor'     -port jmx=6868 -mode cql3 native compression=lz4     -rate threads=100 -pop seq=1..50000000 -log interval=5",
```

Comment:
What's the dataset size?  why not limit the column size by `-col 'size=FIXED(1024) n=FIXED(1)'`?

Reply:
these are the original stress commnads taken out from the 1tb longevity.
They are failing, and we need to test why.